modules/library/data/src/main/java/org/geotools/data/memory/MemoryFeatureReader.java

```diff
@@ -30,20 +34,27 @@
 import org.opengis.feature.simple.SimpleFeatureType;
 
 public class MemoryFeatureReader implements FeatureReader<SimpleFeatureType, SimpleFeature>{
-    ContentState state;
+
     SimpleFeatureType featureType;
     Iterator<SimpleFeature> iterator;
 
     public MemoryFeatureReader(ContentState state, Query query) throws IOException {
```

Comment:
Ha, good question. Thats the next issue I'm going to file. Just want to
separate commits and bugs
Am 14.11.2015 14:53 schrieb "Andrea Aime" <notifications@github.com>:

> In
> modules/library/data/src/main/java/org/geotools/data/memory/MemoryFeatureReader.java
> <https://github.com/geotools/geotools/pull/1044#discussion_r44858726>:
>
> > @@ -30,20 +34,27 @@
> >  import org.opengis.feature.simple.SimpleFeatureType;
> >
> >  public class MemoryFeatureReader implements FeatureReader<SimpleFeatureType, SimpleFeature>{
> > -    ContentState state;
> > +
> >      SimpleFeatureType featureType;
> >      Iterator<SimpleFeature> iterator;
> >
> >      public MemoryFeatureReader(ContentState state, Query query) throws IOException {
>
> Not your fault but... why is the query even passed around?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/geotools/geotools/pull/1044/files#r44858726>.
>